src/qcodes_contrib_drivers/drivers/SPECS/Specs_Tramea.py

```diff
@@ -0,0 +1,750 @@
+from functools import partial
+from typing import Optional
+
+from qcodes import Instrument
+import numpy as np
+import socket
+import nanonis_tramea
+"""
+IMPORTANT
+
+Before using the Driver, you must have our python package installed, because many of functions in this driver are merely
+wrapper functions of functions from said package. How to install:
+
+pip install nanonis-tramea
+"""
+
+
+class NanonisTramea(Instrument):
+    # log = logging.getLogger(__name__)
+
+    def __init__(self, name: str, address: str, port: int, **kwargs):
+        super().__init__(name, **kwargs)
+
+        self._address = address
+        self._port = port
+
+        self.model = "Tramea"
+        self.serial = 1234
+        self.firmware = 1
+
+        self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self._socket.connect((self._address, self._port))
+
+        self.n = nanonis_tramea.Nanonis(self._socket)
+
+        for out in range(1, 51):
+            self.add_parameter(
+                name="{}{}".format("Output", out),
+                set_cmd=partial(self.n.UserOut_ValSet, out),
+                get_cmd=(lambda out=out: self.n.Signals_ValGet(out + 23, 1)[2][0])
+            )
+        for inp in range(24):
+            self.add_parameter(
+                name="{}{}".format("Input", inp+1),
+                get_cmd=(lambda inp=inp: self.n.Signals_ValGet(inp, 1)[2][0])
+            )
+
+
+    def ThreeDSwp_SwpAcqChsSet(self, channelIndexes: np.ndarray[np.integer]):
```

Comment:
```suggestion
    def ThreeDSwp_SwpAcqChsSet(self, channelIndexes: npt.NDArray[np.integer]):
```
To use the single generic argument version of nd array I think you need to use the version from numpy.typing
we usually import that like this

```
import numpy.typing as npt
```

The same fix needs to be applied in a few more places